cypress/Shared/umlsLoginForm.ts

```diff
@@ -0,0 +1,19 @@
+import {CQLEditorPage} from "./CQLEditorPage"
+
+export class umlsLoginForm {
+    //the form, itself, and related buttons and fields
+    public static readonly umlsForm = '[data-testid="UMLS-connect-form"]'
+    public static readonly apiTextInput = '[data-testid="UMLS-key-input"]'
+    public static readonly connectToUMLSButton = '[data-testid="submit-UMLS-key"]'
+    public static readonly umlsConnectSuccessMsg = '[data-testid="UMLS-login-success-text"]'
+    public static readonly closeUMLSForm = '[data-testid="close-UMLS-dialog-button"]'
+    public static readonly genericError = '[data-testid="UMLS-login-generic-error-text"]'
+    public static readonly closeGenericError = '[data-testid="CloseIcon"]'
+    
+    //retrieve API kiey
+    public static retrieveAndEnterAPIKey() : void {
+        cy.readFile('cypress/fixtures/UMLS_API.txt').should('exist').then((fileContents) => {
+            cy.get(this.apiTextInput).type(fileContents)
```

Comment:
the api key already exists as an environment variable. And it needs to be there because it is an ID.
export CYPRESS_VSAC_API_KEY=

Reply:
updated to use environment variable